sunpy/net/dataretriever/sources/tests/test_noaa.py

```diff
@@ -175,6 +176,13 @@ def test_srs_tar_unpack_midyear():
     assert res.data[-1].endswith("20110608SRS.txt")
 
 
+@pytest.mark.remote_data
+def test_srs_missing_tarball():
```

Comment:
What is this testing?

Reply:
Yea I'm not sure the best approach for this test - as of now there is no tarball for 2020 so this test used to fail but not passed. But at some stage I assume noaa will create the 2020 tarball then this test is kinda obsolete.

Reply:
You could make this so it's the current year - 1? Then at least if this code breaks it will fail in Jan 2022 :rofl: 

I think the only sensible way to do this test is mock the responses from the server?

Reply:
I'll see how much trouble mocking the responses will be

Reply:
Done